.travis.yml

```diff
@@ -22,10 +22,10 @@ matrix:
     script: bundle exec rake beaker
     services: docker
     sudo: required
-  - rvm: 2.3.1
+  - rvm: 2.4.0
```

Comment:
2.4.0 is what puppet tests against, and 2.4.0 is available in travis. But yes, we will test on 2.4.1 once we don't have to build it for every test.